client/usr/lib/systemd/system/prometheus-tunnel.service.d/tunnel.conf

```diff
@@ -1,4 +1,4 @@
 [Service]
-Environment=PROMETHEUS_TUNNEL_USER=prometheus-tunnel
-Environment=PROMETHEUS_TUNNEL_HOST=prometheus.example.com
-Environment=PROMETHEUS_TUNNEL_PORT=22
+PROMETHEUS_TUNNEL_USER=prometheus-tunnel
```

Comment:
This isn't required. See the section on `service.d` in https://www.freedesktop.org/software/systemd/man/systemd.unit.html